include/TGUI/Widgets/ChildWindow.hpp

```diff
@@ -199,6 +214,43 @@ namespace tgui
 
 
         /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+        /// @brief Changes the title buttons.
+        ///
+        /// @param buttons  Which buttons should be available in the title bar?
+        ///
+        /// By default ChildWindows only display a close button. You may set the window to show a combination of buttons.
+        /// For example, the following will set the ChildWindow to have both a minimize and close button.
+        /// @code
+        /// childWindow->setTitleButtons(tgui::ChildWindow::TitleButtons::Minimize | tgui::ChildWindow::TitleButtons::Close);
```

Comment:
It might indeed be too verbose but this way the code can be immediately copy-pasted.
It doesn't matter for me whether it is with or without the "tgui::" part though. But if it would be decided to remove it then I would have to change the other places as well where I wrote it like that which is why I am more tempted to just leave it and keep it consistent without effort :smile:.